README.md

```diff
@@ -172,6 +172,13 @@ Configuration is set using a json file stored in `config/config.json` of the bui
 
 An example `config.json` can be seen in the public folder in `config/config.example.json`.
 
+The variables tracked in config/config.json are:
+  * baseUrl(string) - url of the frontend app
+  * clientId (string) - app ide
```

Comment:
Not sure what `app ide` means. Can we please clarify for someone not very familiar with this frontend what these values are to the point that they can run it?